PWGCF/FemtoDream/femtoDreamProducerTask.cxx

```diff
@@ -32,23 +32,21 @@
 #include "Math/Vector4D.h"
 #include "TMath.h"
 
+
 using namespace o2;
 using namespace o2::analysis::femtoDream;
 using namespace o2::framework;
 using namespace o2::framework::expressions;
 
 namespace o2::aod
 {
-using FilteredFullCollision = soa::Filtered<soa::Join<aod::Collisions,
-                                                      aod::EvSels,
-                                                      aod::Mults>>::iterator;
```

Comment:
Why this change?

Reply:
The filtering was done on the Zvtx of the collision and then I could not store all events in the femto collision table as the filtered events are not passed to process(). So I deleted the filtering from the task. The filtering on the Zvtx was actually done additionally in the isSelected() task as well. Now it is only in the isSelected task.

Reply:
Generally, filters are more efficient than to do it in the IsSelected function. So if the trick with checking `ConfIsTrigger` works I would also keep the filter here